### PR TITLE
Updating the ReadMe

### PR DESCRIPTION
**Pull request by ValkyrieSkies**
## What this does
Updates the ReadMe to reflect changes since it was last updated, chiefly that the IRC is dead and Github Desktop is now a much user friendlier git client than SmartGit now. Also includes a quick guide on Github Desktop setup and PR creation seeing as it's so much simpler now.

[administration]
## Changelog

:cl:
 * tweak: Updates the ReadMe on Github

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,32 +4,30 @@
...
+Check the thread on /vg/ for a link or ask a player in-game for an invite.
 
 ---
 
 ### GETTING THE CODE
-The simplest but least useful way to obtain the code is using the Github .zip feature. You can click [here](https://github.com/vgstation-coders/vgstation13/archive/Bleeding-Edge.zip) to get the latest stable code as a .zip file, then unzip it to wherever you want. Alternatively, a much more useful method is to use a git client, the process for getting and using one is described below, (for more information our coders in IRC can tell you how to use one).
+The simplest but least useful way to obtain the code is using the Github .zip feature. You can click [here](https://github.com/vgstation-coders/vgstation13/archive/Bleeding-Edge.zip) to get the latest stable code as a .zip file, then unzip it to wherever you want. This is mostly useful for people looking to get assets from the server or to host a quick short-term game between friends.
 
-### Git client
+Alternatively, the code can be acquired via the use of a Git client as described below. If you're still having difficulty, reach out to one of our coders on Discord and they should be happy to help.
```
Comment (pedr69) [inline README.md:16]: > reach out to one of our coders on Discord 
fuck off

Comment (ValkyrieSkies) [reply README.md:16]: okay where do you intend for them to reach out to them instead

Comment (Securitymain) [reply README.md:16]: Personally, I'd have left out the direct approach and went with something like "ask on discord", but I don't mind either

and pedr is probably just saying FUCK OFF LEAVE ME ALONE

Comment (pedr69) [reply README.md:16]: the thread?
a github discussion?

i'll admit my hatred for "login to see this" may be clouding my judgement but still:
discordites FUCK OFF

**Comment by wotisjanitor** [comment]
Loss of soul

**Comment by Exxion** [comment]
GitHub Desktop is the recommended client? How many people actually use it?

**Comment by ValkyrieSkies** [comment]
> GitHub Desktop is the recommended client? How many people actually use it?

GD is way more intuitive and easy to learn for new coders without lacking any key functionality. Smartgit meanwhile was clunky and obtuse for people not already well acquainted with git.

**Comment by Exxion** [comment]
Yeah SmartGit sucks fucking ass, but how many people use GitHub Desktop as opposed to just VS Code?

**Comment by Securitymain** [comment]
i do

**Comment by Eneocho** [comment]
I just code directly on the browser. Imagine installing any git client. Couldn't be me.

**Comment by Kurfursten** [comment]
Given that I still use CLI, I am giving @Exxion full discretion on this PR. Clearly I am entirely out of touch.

**Comment by Hinaichigo** [comment]
I use Git Bash combined with Git GUI for making it easier to see changes to files and choose files to commit.
Not sure the formal name of them but the ones from here:
https://gitforwindows.org/

**Comment by ValkyrieSkies** [comment]
My other big push for Github Desktop would also be that it's incredibly newbie friendly and thus much easier to onboard new coders with.

**Comment by Eneocho** [comment]
>incredibly newbie friendly

Literally can't be friendlier than coding straight from the browser.

**Comment by Exxion** [comment]
I should clarify that I don't have anything against GitHub Desktop\*, I just wasn't aware that there was a consensus on it
There's still nothing wrong with the Git client built right into VS Code, but you can use whatever works for you

\* I used to use it a long time ago and it sucked back then but I'm sure it's better now

**Comment by ValkyrieSkies** [comment]
> > incredibly newbie friendly
> 
> Literally can't be friendlier than coding straight from the browser.

Very cool eneocho but we're literally talking about the section titled "Git Client"

**Comment by Exxion** [comment]
> Literally can't be friendlier than coding straight from the browser.

You seem to have forgotten about testing

**Comment by Eneocho** [comment]
>testing

"if you code good you need no testing" - Adacovsk (probably)

**Comment by Kurfursten** [comment]
I am still leaving this one up to @Exxion

**Comment by ShiftyRail** [comment]
While I prefer to use git bash, I think github desktop is not bad. The rest of the PR is inconsequential.

**Pull request merged**